OpenIndiana-specific loader color theme

Can be dropped when fix for https://www.illumos.org/issues/10919 is integrated.

diff --git a/usr/src/boot/sys/sys/tem_impl.h b/usr/src/boot/sys/sys/tem_impl.h
index f3e1fd2239..772cf3bcce 100644
--- a/usr/src/boot/sys/sys/tem_impl.h
+++ b/usr/src/boot/sys/sys/tem_impl.h
@@ -120,8 +120,8 @@ typedef uint32_t tem_char_t;	/* 32bit char to support UTF-8 */
  * Default foreground/background color
  */
 
-#define	DEFAULT_ANSI_FOREGROUND	ANSI_COLOR_BLACK
-#define	DEFAULT_ANSI_BACKGROUND	ANSI_COLOR_WHITE
+#define	DEFAULT_ANSI_FOREGROUND	ANSI_COLOR_WHITE
+#define	DEFAULT_ANSI_BACKGROUND	ANSI_COLOR_BLACK
 
 
 #define	BUF_LEN		160 /* Two lines of data can be processed at a time */
